pkg/backup/actions/csi/pvc_action.go

```diff
@@ -252,6 +254,23 @@ func (p *pvcBackupItemAction) Execute(
 		return nil, nil, "", nil, err
 	}
 
+	// Wait until VS associated VSC snapshot handle created before
+	// continue.we later require the vsc restore size
+	vsc, err := csi.WaitUntilVSCHandleIsReady(
+		vs,
+		p.crClient,
+		p.log,
+		backup.Spec.CSISnapshotTimeout.Duration,
+	)
+	if err != nil {
+		p.log.Errorf(
+			"Fail to wait VolumeSnapshot turned to ReadyToUse: %s",
+			err.Error(),
+		)
```

Comment:
```go
p.log.Errorf("Failed to wait for VolumeSnapshot %s/%s to become ReadyToUse within timeout %v: %s", 
        vs.Namespace, vs.Name, backup.Spec.CSISnapshotTimeout.Duration, err.Error())
```
Could help to have this info if the logger doesn't already. If you ran this PR and the log line already contained vs.Name then ignore this.

Reply:
done.added